src/utils/utils.ts

```diff
@@ -0,0 +1,105 @@
+import { cards } from "../cards";
```

Comment:
Love that you abstracted this util funcitons into it's own module, this makes is so some of the business logic is not coupled to the components, hooks, context and it makes them easier to test!